.goreleaser.yaml

```diff
@@ -18,6 +18,18 @@ builds:
       - arm64
       - arm
       - ppc64le
+  - main: ./kubectl/upload/
```

Comment:
I would prefer `kubectl build-remote` ... `kubectl upload` is too broad IMO and might conflict with a future ability to upload a Dataset itself rather than the container for loading a Dataset... and longer term this plugin should only be registered for the following to avoid taking up too much kubectl plugin real estate (we should not assume Substratus is the only thing the end user is using k8s for).

```
kubectl build-remote model
kubectl build-remote server
kubectl build-remote notebook
kubectl build-remote dataset
```

Reply:
Yeah, I should have commented that I wasn't totally satisfied with `kubectl upload` - it's just too generic. The plugin started as `upload-kind` and became `upload-resource` prior to the current iteration.

Naming is hard but `build-remote` describes the high-level intent of the process better than any of the above. I'll go with that 👍 

I was actually okay punting on a less-than-ideal name here because medium-term I think we should get away from kubectl plugins and migrate toward a `substratus` or `ssai` CLI that rolls in these utilities. I'll capture that as an issue.

Reply:
Captured here: https://github.com/substratusai/substratus/issues/120